help_to_heat/frontdoor/views.py

```diff
@@ -746,8 +746,7 @@ def save_post_data(self, data, session_id, page_name):
         if country == country_field_scotland and uprn is not None:
             epc = interface.api.epc.get_epc_scotland(uprn)
             if epc != {}:
-                epc_details = {"current-energy-rating": epc.get("rating"), "lodgement-date": epc.get("date")}
-                data[epc_details_field] = epc_details
```

Comment:
Would prefer to just pass the fields we care about to the template, while we're changing this

Reply:
for E&W? in the scotland case here all the fields passed here are ones we care about

Reply:
Ah sorry, I had this backwards - never mind